packages/app-store/office365video/lib/VideoApiAdapter.test.ts

```diff
@@ -0,0 +1,282 @@
+import prismaMock from "../../../../tests/libs/__mocks__/prismaMock";
```

Comment:
We should add similar tests for all integrations as and when we add OAuthManager to them